tests/manage/pv_services/test_cephfs_with_mds_network_failure.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+from random import randint
+from ocs_ci.ocs import constants
+from threading import Thread
+from ocs_ci.ocs.cluster import get_mds_standby_replay_info
+from ocs_ci.helpers.helpers import disable_vm_network_for_duration
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required, tier1
+from ocs_ci.ocs.resources.pod import search_pattern_in_pod_logs
+
+log = logging.getLogger(__name__)
+
+
+@vsphere_platform_required
+class TestCephFSWithMDSNetworkFailure:
+    @tier1
+    def test_cephfs_network_interruption_standby_replay_MDS(self, pod_factory):
+        """Test MDS crash by causing network interruption to the standby-replay daemon
+
+        1. Deploy vmware upi cluster based on the test environment
+        2. Identify the nodes running active and standby MDS by running commands like "ceph fs status"
+        3. Run the cephFS IOs, if not running
+        5. Interrupt the network on standby-replay MDS node by running command "ifconfig <network-device> down"
```

Comment:
You can remove this as we are disabling network from vpshere console. Also correct the step number